CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
```

Comment:
Should we keep this as CC or change it to Apache 2.0 as well as the code?

Reply:
Documentation goes under CC normally, https://wiki.hyperledger.org/display/TSC/Copyright+and+License+Policy